### PR TITLE
VE-74 Marine Regions package

### PR DESCRIPTION
**Pull request by rdgfuentes**
- Added marine-regions npm package to get eez, mpa and rfmo
- Populate eez, mpa and rfmo regions in vessel examiner dropdowns
- Fixed some settings translations

### CODE REVIEW

File: applications/vessel-history/src/features/settings/Settings.tsx [hunk 1]
```diff
@@ -59,7 +70,10 @@ const Settings: React.FC = (): React.ReactElement => {
           onClick={onBackClick}
           className={styles.backButton}
         />
-        <h1>{t('settings.title',"Settings")} {selectedOption && <span> - {optionsData[selectedOption].title}</span>}</h1>
+        <h1>
+          {t('settings.title', 'Settings')}{' '}
```
Comment (fertandil87) [inline applications/vessel-history/src/features/settings/Settings.tsx:74]: {' '} react not understand a simple space? :hankey:

**Review by fertandil87 — APPROVED** [review]

File: applications/vessel-history/src/features/settings/Settings.tsx (hunk 1 above)
Comment (rdgfuentes) [reply applications/vessel-history/src/features/settings/Settings.tsx:74]: 🤦 This was fixed by the auto-formatter on save 🤷‍♂️

File: packages/marine-regions/src/data/eez_areas.query.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (j8seangel) [inline packages/marine-regions/src/data/eez_areas.query.sql:1]: 🙇 good idea to store this so next time will be easier

**Comment by j8seangel** [comment]
Great work!

**Pull request merged**